src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,97 @@
-export const Pagination = () => {};
+import { FC, useEffect, useState } from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const [chosenPage, setChosenPage] = useState(currentPage);
```

Comment:
why do we need this useState if we can directly call onPageChange?

Reply:
isn't fixed. You shouldn't use an additional state, you can just use onPageChange function directly at Pagination.tsx